deepmd/dpmodel/fitting/polarizability_fitting.py

```diff
@@ -174,7 +175,9 @@ def serialize(self) -> dict:
         data["embedding_width"] = self.embedding_width
         data["old_impl"] = self.old_impl
         data["fit_diag"] = self.fit_diag
+        data["shift_diag"] = self.shift_diag
         data["@variables"]["scale"] = self.scale
+        data["@variables"]["constant_matrix"] = self.constant_matrix
```

Comment:
We may need to bump the version, considering new serialized data can be not be deserialized by the old code.

Reply:
you mean model plugin?

Reply:
I mean the version system introduced in #3349.